internal/controller/authenticator/jwtcachefiller/jwtcachefiller.go

```diff
@@ -569,5 +606,8 @@ func (c *jwtCacheFillerController) updateStatus(
 		return nil
 	}
 	_, err := c.client.AuthenticationV1alpha1().JWTAuthenticators().UpdateStatus(ctx, updated, metav1.UpdateOptions{})
+	if err != nil {
+		c.log.Info(fmt.Sprintf("ERROR: %v", err))
+	}
```

Comment:
The arguments to the `Info` method are not done correctly here. Also, do you even need this at all? Won't the error be logged anyway because it is returned by the Sync function?

Reply:
Yeah I did the opposite, should have deleted it from WebhookCachefiller Sync, instead of adding here, so they would be consistent.  Will update both & remove.